Blogpost about comment locking and report categories

krauselukas opened the pull request.

eduardoj commented on _posts/development/2023-10-18-comment-locking-and-report-categories.md:
```diff
@@ -0,0 +1,42 @@
+
+There can be several reasons for not wanting comments on specific projects, packages
+or requests. A very common one is simply to shift the daily conversations to a dedicated place
+that is better suited for this purpose. Until now the only option you had is to leave
```
```suggestion
that is better suited for this purpose. Until now, the only option you had is to leave
```

eduardoj commented on _posts/development/2023-10-18-comment-locking-and-report-categories.md:
```diff
@@ -0,0 +1,42 @@
+There can be several reasons for not wanting comments on specific projects, packages
+or requests. A very common one is simply to shift the daily conversations to a dedicated place
+that is better suited for this purpose. Until now the only option you had is to leave
+a friendly comment with a hint. From now on you can simply lock the commenting
```
```suggestion
a friendly comment with a hint. From now on, you can simply lock the commenting
```

eduardoj commented on _posts/development/2023-10-18-comment-locking-and-report-categories.md:
```diff
@@ -0,0 +1,42 @@
+that is better suited for this purpose. Until now the only option you had is to leave
+a friendly comment with a hint. From now on you can simply lock the commenting
+functionality directly from the comment section. As a maintainer you still
+will be able to leave a comment, for everyone else it will be locked.
```
```suggestion
will be able to leave a comment. For everyone else it will be locked.
```

eduardoj commented on _posts/development/2023-10-18-comment-locking-and-report-categories.md:
```diff
@@ -0,0 +1,42 @@
+
+The reason for reporting a project, package, user or comment are often similar.
+Comments often fall under the category spam, packages might be submitted with
+a forbidden license. To avoid that you have to repeat yourself, we introduced
```
```suggestion
a forbidden license, ... To avoid that you have to repeat yourself, we introduced
```

eduardoj reviewed (changes requested):
I left a couple of suggestions, mostly related to punctuation.

I also suggest you write every sentence in a different line: it would make no difference when rendering the page, but it will make easier to review the text.

saraycp commented on _posts/development/2023-10-18-comment-locking-and-report-categories.md:
```diff
@@ -0,0 +1,42 @@
+introducing comment locks. On top of this we enhanced the reporting feature by a
+a set of categories to ease the submission.
+
+All of the following is part of the *Content Moderation* [beta program](/2018/10/04/the-beta-program/).
```
This is the second blog post regarding Moderation. I'd say it's time to _include_ the paragraph mentioning all the blog posts related to the topic. As we do for other opportunities.

krauselukas replied:
true, will do so

saraycp commented on _posts/development/2023-10-18-comment-locking-and-report-categories.md:
```diff
@@ -0,0 +1,42 @@
+
+<figure>
+  <img src="/images/posts/screenshot_comment_lock.png" alt="Screenshot of a locked comment section" />
+  <figcaption>Locked comment section from maintainer perspective</figcaption>
```
It's important people know that what they see in the image is only for maintainers. It's true that's clarified in the `figcaption` but it can go unnoticed. What about adding it as normal text right before the image?

krauselukas commented:
@eduardoj @saraycp @hellcp-work please have another look, I addressed your reviews

eduardoj commented:
> @eduardoj @saraycp @hellcp-work please have another look, I addressed your reviews

I still miss this suggestion: "I also suggest you write every sentence in a different line: it would make no difference when rendering the page, but it will make easier to review the text."

krauselukas commented:
> > @eduardoj @saraycp @hellcp-work please have another look, I addressed your reviews
> 
> I still miss this suggestion: "I also suggest you write every sentence in a different line: it would make no difference when rendering the page, but it will make easier to review the text."

@eduardoj I overlooked it, its now applied

krauselukas commented:
@saraycp done :)
